codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -276,12 +277,21 @@ import Unison.Sqlite
 
 -- * main squeeze
 
+currentSchemaVersion :: SchemaVersion
+currentSchemaVersion = 7
+
 createSchema :: Transaction ()
 createSchema = do
   executeFile [hereFile|unison/sql/create.sql|]
+  execute insertSchemaVersionSql (Only currentSchemaVersion)
```

Comment:
?

Reply:
I moved the insert from `create.sql` into here so that I could use the same constant in the Haskell code to avoid them going out of sync.